ui/src/api.js

```diff
@@ -105,6 +105,26 @@ export async function getCenterRegion(sceneId, x, y, w, h) {
   return minRegion;
 }
 
+export async function getRegionClosestTo(sceneId, x, y, w, h, rx, ry) {
+  const regions = await getRegions(sceneId, x, y, w, h);
```

Comment:
This should be replaced by a purpose-built API call. It's too expensive this way.